website/docs/components/app-header/partials/guidelines/overview.md

```diff
@@ -0,0 +1 @@
+The App Header provides consistent controls for navigating between global areas of the application (organizations and projects) and accessing application utilities like support, documentation, and user settings. It should be paired with the Side Nav component within the App Frame (in code).
```

Comment:
Should we link to the SideNav and AppFrame pages?

Reply:
We should! I've added these